[OSDOCS-7645]: Remove Note Regarding Inability to Delete Default Machine Pool, other MP updates (#64189)
openshift/openshift-docs: modules/rosa-adding-taints-cli.adoc

You can add taints to a machine pool for your Red Hat OpenShift Service on AWS (ROSA) cluster by using the ROSA CLI.

[NOTE]
====

> For users of ROSA CLI `rosa` version 1.2.25 and prior versions, the number of taints cannot be changed within the machine pool (ID=`Default`) created along with the cluster. cannot be deleted. For users of ROSA CLI `rosa` version 1.2.26 and beyond, the number of taints can be changed within the machine pool (ID=`worker`) created along with the cluster.

Reviewer: ```suggestion
For users of ROSA CLI `rosa` version 1.2.25 and prior versions, the number of taints cannot be changed within the machine pool (ID=`Default`) created along with the cluster cannot be deleted. For users of ROSA CLI `rosa` version 1.2.26 and beyond, the number of taints can be changed within the machine pool (ID=`worker`) created along with the cluster.
```

> For users of ROSA CLI `rosa` version 1.2.25 and prior versions, the number of taints cannot be changed within the machine pool (ID=`Default`) created along with the cluster. For users of ROSA CLI `rosa` version 1.2.26 and beyond, the number of taints can be changed within the machine pool (ID=`worker`) created along with the cluster. There must be at least one machine pool without any taints and with at least two replicas for a single-AZ cluster or three replicas for a multi-AZ cluster.

Reviewer 2: ```suggestion
For users of ROSA CLI `rosa` version 1.2.25 and earlier versions, the number of taints cannot be changed within the machine pool (ID=`Default`) that are created along with the cluster. For users of ROSA CLI `rosa` version 1.2.26 and later, the number of taints can be changed within the machine pool (ID=`worker`) that are created along with the cluster. There must be at least one machine pool without any taints and with at least two replicas for a single-AZ cluster or three replicas for a multi-AZ cluster.
```
